docs/administration/user-permissions.md

```diff
@@ -0,0 +1,99 @@
+
+Every drupal user is mapped to a CiviCRM contact. To assign a CiviRemote role, open the summary of the CiviCRM contact and edit the field set **RemoteContact Information**.
+
+- for reviewers add the role **CiviRemote User**.
+- for receiving contacts add the roles **CiviRemote User** and **CiviRemote Funding**.
+
+!!! question
+    It can be handy to allow the creation of Drupal user accounts for unauthenticated users, for example new applicants. This is done at *Arbeit und Leben*, I think. How is this done exactly? I want to add this as a tip.
+
+!!! note
+    The CiviRemote User Roles are automatically synchronised with the Drupal user account during login. If there are problems with the synchronisation, open the drupal user overview at `admin/people`. Select the user that you want to assign the roles to and perform the action **CiviRemote: Match contact(s)** and afterward **CiviRemote: Synchronise CiviRemote roles**.
+
+
+## Funding Program
+
+After creating a funding program, it will be listed at **Funding > Funding Programs**. There are several actions available, namely **Edit Recipients**, **Edit permissions** and **Edit initial funding case permissions**. These three settings have to be configured correctly before opening the program for applicants.
+
+### Edit Recipients
+
+When an applicant creates an application, one field of the application form asks who is going to be the recipient of the funding. The recipient is not necessarily identical to the applicant. The field in the application form is a dropdown menu where the correct recipient needs to be chosen. The values that are shown in the dropdown menu need to be configured at **Edit Recipients**.
+
+As an example, consider a funding program that funds organizations. The application is not created by the organization but by an employee of that organization. In this case, the applicant is the employee and the recipient is the organization. If both exist as contacts in CiviCRM and are connected by a relationship of type _Employee_, the configuration would look as follows:
+
+![](../img/permissions_edit_recipients.png)
+
+The dropdown menu will then show all organizations that have an _Employee_ relationship with the applicant that is logged in and creating the application.
+
+### Edit permissions
+
+These permissions are related to a funding program, independently of a specific funding case or application.
+
+The following permissions exist:
+
+* **Application: create** allows contacts to create a new application for this funding program.
+* **Application: apply** allows contacts to create a new application and apply it in one step.
+* **View** specifies which funding programs are available to create an application for. It also specifies which funding programs are visible in the CiviCRM funding program list.
```

Comment:
I also removed the sentence "Both implicitly include the permission **View**." that comes directly below. I note that this is the sentence that confuse(d) me and why we had a discussion above. I think it can be confusing to the reader as well and the information does not seem to be very relevant.